components/event/org.wso2.carbon.event.ui/src/main/java/org/wso2/carbon/event/ui/UIUtils.java

```diff
@@ -35,54 +68,134 @@ public static RemoteAuthorizationManagerServiceStub getAuthManagementClient(Serv
+
+    /**
+     * Returns the user admin client for user handling
+     * Suppressing warning of unused declaration as it used by the UI (JSP pages)
+     *
+     * @param config the servlet configuration
+     * @param session the http session
+     * @param request the http servlet request
+     * @return the user admin client
+     * @throws Exception
+     */
+    @SuppressWarnings("UnusedDeclaration")
+    public static UserAdminClient getUserAdminClient(ServletConfig config, HttpSession session,
+                                                     HttpServletRequest request) throws Exception {
         String backendServerURL = CarbonUIUtil.getServerURL(config.getServletContext(), session);
-        if(!backendServerURL.endsWith("/")){
+        if (!backendServerURL.endsWith("/")) {
             backendServerURL = backendServerURL + "/";
         }
         String cookie = (String) session.getAttribute(ServerConstants.ADMIN_SERVICE_COOKIE);
-        return new UserAdminClient(cookie, backendServerURL,null);
+        return new UserAdminClient(cookie, backendServerURL, null);
     }
-    
-    public static String getValue(HttpServletRequest request,String name){
+
+    /**
+     * Gets values of a parameter in a HttpServletRequest
+     * Suppressing warning of unused declaration as it used by the UI (JSP pages)
+     *
+     * @param request the HttpServletRequest
+     * @param name    parameter name
+     * @return values of parameter
+     */
+    @SuppressWarnings("UnusedDeclaration")
+    public static String getValue(HttpServletRequest request, String name) {
         String value = request.getParameter(name);
-        if("null".equals(value)){
+        if ("null".equals(value)) {
             return "";
         }
-        return value != null? value : "";
+        return value != null ? value : "";
     }
-    
-    public static String getDialog(String message){
-        return "CARBON.showErrorDialog('"+message+"');";
+
+    /**
+     * Gets error dialog
+     * Suppressing warning of unused declaration as it used by the UI (JSP pages)
+     *
+     * @param message the message to show in error
+     * @return the error alert
+     */
+    @SuppressWarnings("UnusedDeclaration")
+    public static String getDialog(String message) {
+        return "CARBON.showErrorDialog('" + message + "');";
     }
-    
-    public static String getSubscriptionMode(String serverMode){
-         if (serverMode.equals(UIConstants.SUBSCRIPTION_MODE_1)){
-             return UIConstants.SUBSCRIPTION_MODE_1_DESCRIPTION;
-        }else if(serverMode.equals(UIConstants.SUBSCRIPTION_MODE_2)){
+
+    /**
+     * Gets subscription mode description
+     * Suppressing warning of unused declaration as it used by the UI (JSP pages)
+     *
+     * @param serverMode subscription mode
+     * @return subscription mode description
+     */
+    @SuppressWarnings("UnusedDeclaration")
```

Comment:
Its being used by the JSP pages. IntelliJ gives out a warning stating that it is not being used. Therefore I have suppressed it.